docker/images/files/config.js

```diff
@@ -5,7 +5,7 @@
 //var penpotAllowDemoUsers = <true|false>;
 //var penpotGoogleClientID = "<google-client-id-here>";
 //var penpotGitlabClientID = "<gitlab-client-id-here>";
-//var penpotGithubClientID = "<github-client-id-here>";
+//var penpotGitHubClientID = "<github-client-id-here>";
```

Comment:
This change can break thinks, I think we can live with Github, and not GitHub

Reply:
Ok, will drop